test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -259,7 +259,7 @@
 (deftest ^:integration static-file-content-endpoint-test-no-code-content-command
   (logging/with-test-logging
    (testing "the /static_file_content endpoint errors if :code-content-command is not set"
-     (bootstrap/with-puppetserver-running
+     (bootstrap/with-puppetserver-running-with-mock-jrubies
```

Comment:
Are you asking for a docstring in the defmacro itself or a comment at each point where the macro is used to say why it's safe to use a mock instead of the real thing - or both?